multiqc/modules/mosdepth/mosdepth.py

```diff
@@ -209,31 +273,18 @@ def __init__(self):
         self.general_stats_addcols(genstats, genstats_headers)
 
     def parse_cov_dist(self):
-        genstats = defaultdict(OrderedDict)  # mean coverage
-        cumcov_dist_data = defaultdict(OrderedDict)  # cumulative distribution
-        cov_dist_data = defaultdict(OrderedDict)  # absolute (non-cumulative) coverage
+        genstats = defaultdict(dict)  # mean coverage
+        cumcov_dist_data = defaultdict(dict)  # cumulative distribution
+        cov_dist_data = defaultdict(dict)  # absolute (non-cumulative) coverage
```

Comment:
There's a lot of this.. I've been thinking of stripping out `OrderedDict` throughout the entire repo, but it's a bit scary in case it breaks stuff. One for a separate PR anyway..

Reply:
Yup 👍🏻  I already have an issue for this here: https://github.com/ewels/MultiQC/issues/1919